app/controlplane/api/workflowcontract/v1/crafting_schema.proto

```diff
@@ -168,7 +168,11 @@ message PolicyAttachment {
   map<string, string> with = 5;
 
   // List of requirements this policy contributes to satisfy
-  repeated string requirements = 6;
+  repeated string requirements = 6 [(buf.validate.field).repeated.items = {
+    string: {
+      pattern: "^[^\\s@]+(@[^\\s@]+)?$", // whitespaces not allowed, lock is optional (@)
```

Comment:
I know that I said we should force it but now I am wondering if we should be lax by default as you did. What do you think?

Reply:
Yes, but at least I would force the `@` for the version separator (or `/`, or `:`, anything), as we do in other places

Reply:
I've updated the rule to have a mandatory requirement prefix for grouping them into frameworks, groups, or similar. `myframework/myrequirement@version`